src/audio/copier/copier.c

```diff
@@ -1442,6 +1445,24 @@ static int copier_set_sink_fmt(struct comp_dev *dev, const void *data,
 							      &sink_fmt->sink_fmt, ipc4_gtw_none,
 							      ipc4_bidirection);
 
+	/* update corresponding sink format */
+	list_for_item(sink_list, &dev->bsink_list) {
+		int sink_id;
+
+		sink = container_of(sink_list, struct comp_buffer, source_list);
+		sink_c = buffer_acquire(sink);
+
+		/* only updated current used sink buffer */
```

Comment:
nit-pick, this comment is not needed. the first comment to "update corresponding sink format" is enough

Reply:
sure, thanks